service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/PopulateStandardDirectionsHandlerTest.java

```diff
@@ -68,12 +70,15 @@
     @Autowired
     private HearingBookingService hearingBookingService;
 
+    @MockBean
+    private UserDetailsService userDetailsService;
+
     @Autowired
     private ObjectMapper objectMapper;
 
     private PopulateStandardDirectionsHandler populateStandardDirectionsHandler;
 
-    private DirectionHelperService directionHelperService = new DirectionHelperService();
+    private DirectionHelperService directionHelperService = new DirectionHelperService(userDetailsService);
```

Comment:
Not sure but I think use can use `@InjectMocks` instead of `= new ...`

Reply:
I can do this in a few places 👍